.github/workflows/integrate.yaml

```diff
@@ -15,55 +15,21 @@ on:
       - "flake.lock"
       - "agora.cabal"
 jobs:
-  check-formatting:
+  Build:
```

Comment:
I would prefer kebab-case for this. Maybe just `flake`? This will give `flake (.formatCheck)`, for example.

Reply:
I'm fixing this. For some reason CI refuses to work again...